tests/test_bible.py

```diff
@@ -27,6 +30,11 @@ def test_includes(self):
         with self.assertRaises(RangeError):
             # acts 2 has 47 verses
             self.two_books.includes(bible.Verse('Acts 2:48'))
+        
+""" TODO: test for range of boxed/omitted passages        
+        # with self.assertRaises(RangeError):
+        #     # mark 16:9-20 has been boxed/omitted
+        #     self.boxed.includes(bible.Verse('Mark 16:9')) """
```

Comment:
Why is this commented out and a TODO?

Reply:
Note to myself to implement RangedError exception class later on